ros2_batch_job/__main__.py

```diff
@@ -204,6 +204,9 @@ def get_args(sysargv=None):
     parser.add_argument(
         '--force-ansi-color', default=False, action='store_true',
         help="forces this program to output ansi color")
+    parser.add_argument(
+        '--ros-distro', default='foxy',
```

Comment:
If this is always being passed can the default be removed and made required instead?

Reply:
Done in 24523b7